sql/dql/basic-selects/select-cols.md

```diff
@@ -28,19 +28,40 @@ aspects:
 ---
 ## Content
```

Comment:
this seems pretty much similar to: `sql/dql/basic-queries/select.md`

which one comes first? I assume this one? if so you might want to change the focus on the `sql/dql/basic-queries/select.md` to be on aliasing

Reply:
this is what I aimed to do.
- in basic-selects/select-cols.md -> introduce how to select specific columns and why it is not a good idea to select * from a table
- in basic-queries/select.md -> remind the user how to select all and select specific columns, introduce aliases `AS`